client/jetpack-onboarding/steps/contact-form.jsx

```diff
@@ -65,44 +63,33 @@ class JetpackOnboardingContactFormStep extends React.PureComponent {
 	}
 
 	renderActionTile() {
-		const { hasContactForm, isConnected, siteUrl, translate } = this.props;
-		const headerText = translate( "Let's grow your audience with Jetpack." );
-		const subHeaderText = (
-			<Fragment>
-				{ translate(
-					'A great first step is adding a Contact Us page that includes a contact form.'
-				) }
-				<br />
-				{ translate( 'Create a Jetpack account to unlock this and dozens of other features.' ) }
-			</Fragment>
-		);
-		const connectUrl = addQueryArgs(
-			{
-				url: siteUrl,
-				// TODO: add a parameter to the JPC to redirect back to this step after completion
-				// and in the redirect URL include the ?action=add_contact_form parameter
-				// to actually trigger the page and form creation action after getting back to JPO
-			},
-			'/jetpack/connect'
+		const { hasContactForm, siteId, translate } = this.props;
+		const header = (
+			<FormattedHeader
+				headerText={ translate( "Let's grow your audience with Jetpack." ) }
+				subHeaderText={
+					<Fragment>
+						{ translate(
+							'A great first step is adding a Contact Us page that includes a contact form.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'A great first step is adding a Contact Us page that includes Jetpack’s contact form.' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).